pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -486,6 +490,29 @@
   ai_resources_desc_03: "Supports integration of Artificial Intelligence knowledge throughout K-12. Offer free professional development, webinars, conference presentations, and peer-to-peer learning opportunities."
   ai_resources_btn_aria_label_03: "Explore the AI for Teachers website"
 
+  ai_pl_101_hero_heading: "AI 101 for Teachers"
+  ai_pl_101_hero_desc: "Discover the transformative potential of artificial intelligence (AI) in education with our free online learning series for educators."
+  ai_pl_101_hero_presented: "Presented by"
+  ai_pl_101_intro_heading: "What is AI 101 for Teachers?"
+  ai_pl_101_intro_text: "This is a comprehensive online learning series for any teacher and educator interested in the groundbreaking world of artificial intelligence (AI) and its transformative potential in education. Partners Code.org, Khan Academy, ISTE, ETS, and TeachAI are offering engaging sessions with renowned experts that will demystify AI, explore responsible implementation, address bias, and showcase how AI-powered learning can revolutionize student outcomes. Join us on this journey of exploration and empowerment, and unlock the future of teaching with and about AI."
+  ai_pl_101_partnership_between: "A partnership between"
+  ai_pl_101_join_email_title: "Sign up for future AI updates"
+  ai_pl_101_pro_join_email_desc: "Be the first to hear about new AI opportunities from Code.org, Khan Academy, ISTE, and ETS."
+  ai_pl_101_explore_title: "Explore additional AI resources"
+  ai_pl_101_explore_desc: "We have a variety of additional resources for  you from Code.org, ETS, ISTEI, Khan Academy and TeachAI to support teachers in this new and growing field. Check them out below."
```

Comment:
Nit: could you remove the double space between "for" and "you" in this string please